servers/zts/src/main/java/com/yahoo/athenz/zts/utils/ZTSUtils.java

```diff
@@ -305,7 +311,53 @@ public static Identity generateIdentity(InstanceCertManager certManager, final S
     }
 
     public static SSLContext createServerClientSSLContext(PrivateKeyStore privateKeyStore) {
+        if (Boolean.parseBoolean(System.getProperty(ATHENZ_PROP_USE_LOCAL_CERT_AND_KEY_FOR_SSL, ATHENZ_DEFAULT_USE_LOCAL_CERT_AND_KEY_FOR_SSL))) {
+            return getSslContextAthenzCert(privateKeyStore);
+        } else {
+            return getSslContextExternalCert(privateKeyStore);
+        }
+    }
+
+    private static SSLContext getSslContextAthenzCert(PrivateKeyStore privateKeyStore) {
```

Comment:
this should be called getSslContextUsingKeyCertFiles

Reply:
Fixed